### PR TITLE
Addition of the description attribute in the payload

### PR DESCRIPTION
**Pull request by shrsr**

### CODE REVIEW

**Review by samiib — APPROVED** [review]
LGTM

File: models/schema_template_contract_filter.go [hunk 1]
```diff
@@ -18,6 +18,10 @@ func NewTemplateContractFilterRelationShip(ops, path, action, priority string, f
 		filterMap["priorityOverride"] = priority
 	}
 
+	if desc != "" {
```
[the remark this answers is not in the document]
Comment (shrsr) [reply models/schema_template_contract_filter.go:21]: I was not sure if description is supported in all the versions? Especially after looking at the other attributes.

[the remark this answers is not in the document]
Comment (akinross) [reply models/schema_template_contract_filter.go:21]: Ok but should be not add the check then for all descriptions? Or you are sure on those? Also I think when not supported attribute ndo still allows us to write the variable unless there is validation on that attribute

[the remark this answers is not in the document]
Comment (shrsr) [reply models/schema_template_contract_filter.go]: Removing the condition after discussion. Non-existent attributes will be ignored in the payload for NDO

[the remark this answers is not in the document]
Comment (shrsr) [reply models/schema_template_contract_filter.go]: Removing the condition after discussion

**Review by sajagana — APPROVED** [review]
LGTM!

**Review by akinross — APPROVED** [review]
LGTM

**Review by lhercot — APPROVED** [review]
LGTM

**Pull request merged**